src/DIRAC/DataManagementSystem/scripts/dirac_dms_filecatalog_cli.py

```diff
@@ -20,50 +20,66 @@
 
   FC:/>
 """
-import sys
 
 from DIRAC.Core.Base.Script import Script
 
 
 @Script()
 def main():
-    fcType = "FileCatalog"
+    fcType = None
+    catalog = None
     Script.registerSwitch("f:", "file-catalog=", f"   Catalog client type to use (default {fcType})")
```

Comment:
What is the default now that `fcType = None` ?

Reply:
If the fcType is None, the usual logic of discovering the appropriate catalog defined for the VO to which the user belongs applies. The user's VO is determined based on the user's proxy information.

Reply:
Sorry for not being clear in this case: the docstring for the switch should be adapted `default {fcType}`

Reply:
Ah yes, you are right